CMAKE Policy Error CMP0074 Windows 10, PCL 1.9.1, on PCL_ROOT

Receiving CMP0074 warning on the PCL_ROOT variable and it is reported as ignored.  I'm trying to build the project in the "Using PCL in your own project".  I've installed with the all in one installer and I've followed the issue in the link below but it was reported as fixed.

https://github.com/PointCloudLibrary/pcl/issues/2425

I'm very much a subject programmer and manipulation of cmake is not my expertise, I'm not married to using the particular PCL version, so willing to accept reverting to a prior known compatible CMAKE - PCL versions  as a fix.  Thanks!

##  Environment
* Windows 10 Version 1709, 64 bit
* Compiler: msvc 2017
* PCL Version: 1.9.1, installed via PCL-1.9.1-AllInOne-msvc2017-win64.exe
* CMAKE Version: 3.13.2

##  Outputs
[CMakeError.log](https://github.com/PointCloudLibrary/pcl/files/2764562/CMakeError.log)
[CMakeOutput.log](https://github.com/PointCloudLibrary/pcl/files/2764568/CMakeOutput.log)
[command_line_output.txt](https://github.com/PointCloudLibrary/pcl/files/2764569/command_line_output.txt)

## Potential Conflict Source
When installing PCL 1.9.1 I had the error message regarding too long of environmental variable name so manually made the following changes
* Added C:\Program Files\PCL 1.9.1\bin to Path Variable
* Added PCL_ROOT variable as C:\Program Files\PCL 1.9.1

## Possible Solution
- Go to previous version of PCL or CMAKE?

So removed the boost variables in the environmental variables and it seems to be drawing from the proper Boost in the 3rd party library install.  The cmake error with pthread.h remains
* fatal error C1083: Cannot open include file: 'pthread.h': No such file or directory

As well as the linking error for the boost library when going to build
* pcd_write.obj : error LNK2019: unresolved external symbol "class boost::system::error_category const & __cdecl boost::system::detail::generic_category_ncx(void)" (?generic_category_ncx@detail@system@boost@@YAAEBVerror_category@23@XZ) referenced in function "class boost::system::error_category const & __cdecl boost::system::generic_category(void)" (?generic_category@system@boost@@YAAEBVerror_category@12@XZ)

When I searched for a pthread file in the directory of pcl 1.9.1 this is what I find.
![image](https://user-images.githubusercontent.com/15371018/51398217-068feb80-1b11-11e9-88dc-5ee9fbaf3caf.png)

# Suggestions
* I do have some environmental variables pointing to a separate QT directory for building a different library, and a FLANN_ROOT from earlier install attempt through attempts at but I doubt that is involved.  If requested I can remove those and try to rebuild.

--- a/doc/tutorials/content/using_pcl_pcl_config.rst
+++ b/doc/tutorials/content/using_pcl_pcl_config.rst
@@ -28,7 +28,7 @@ CMakeLists.txt that contains:
    link_directories(${PCL_LIBRARY_DIRS})
    add_definitions(${PCL_DEFINITIONS})
    add_executable(pcd_write_test pcd_write.cpp)
-   target_link_libraries(pcd_write_test ${PCL_COMMON_LIBRARIES} ${PCL_IO_LIBRARIES})
+   target_link_libraries(pcd_write_test ${PCL_LIBRARIES})
 
 The explanation
 ---------------
@@ -97,7 +97,7 @@ Windows platform and blank on UNIX) and the permissions.
 
 .. code-block:: cmake
 
-   target_link_libraries(pcd_write_test ${PCL_COMMON_LIBRARIES} ${PCL_IO_LIBRARIES})
+   target_link_libraries(pcd_write_test ${PCL_LIBRARIES})
 
 The executable we are building makes call to PCL functions. So far, we
 have only included the PCL headers so the compilers knows about the

